### PR TITLE
Add separable/depthwise convolution, data augmentation and multiple GPU support

### PR DESCRIPTION
**Pull request by DeeperMind**
This pull request added separable convolution and depthwise convolution to the operation library of the training container for reinforcement learning suggestion. It also added data augmentation and multiple GPU support.

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/katib/393)

### CODE REVIEW

**Comment by DeeperMind** [comment]
\assign @hougangliu @johnugeorge

**Comment by DeeperMind** [comment]
recheck

File: examples/ENAS-example.yaml [hunk 1]
```diff
@@ -0,0 +1,117 @@
+# This example reproduces the search space of Efficient Neural Architecture Search via Parameter Sharing
+# Is capable of achieving better result and lower generaliation error than the full search space
+# For more details, please visit https://arxiv.org/abs/1802.03268
+
+# In practice, setting up a limited search space like this is more likely to get better performance. 
+
+apiVersion: "kubeflow.org/v1alpha1"
+kind: StudyJob
```
[the remark this answers is not in the document]
Comment (DeeperMind) [reply examples/ENAS-example.yaml]: The ENAS example has been deleted. I added some comments in the original example instead.

**Comment by andreyvelich** [comment]
@YujiOshima @hougangliu @johnugeorge 
I think this PR is ready to merge. Please, can you review it?

**Pull request merged**